manifests/filter/header_user.pp

```diff
@@ -0,0 +1,66 @@
+# == Resource: repose::filter::header_user
+#
+# This is a resource for generating header-user configuration files
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool. Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String. Config filename
+# Defaults to <tt>header-user.cfg.xml</tt>
+#
+# [*source-headers*]
+# List containing source headers with the ids of the source headers and their
```

Comment:
is this a list or a hash ? or a list of hashes ?  Can we add a detailed example like in https://github.com/rackerlabs/puppet-repose/blob/master/manifests/filter/system_model.pp

Reply:
ah list of hashes, thanks

Reply:
addressed.